### ISSUE TITLE
Feature: 1.16 - Fromimage mode that allows the painting of biome groups

### ISSUE THREAD

**Issue by BiomeBundle** [PG85/OpenTerrainGenerator#751]
New feature:

Ability to assign hex colours to biome groups, and then paint them in fromimage mode and allow random generation to take over from there.

Would allow a simpler, more user friendly way to paint landmasses, climate zones and oceans.

**Comment by NLBlackEagle** [comment]
I double this suggestion, this would allow fromimage parts to enhance random mode or vice versa as well as offer the following features;

- Fix the jagged edges, cut off mountains and so on between continuenormal and fromimage (By assigning a biomegroup as border biomes)
- The ability to use mirrormode on a biomegroup painted map so that hot zones never border cold zones (this is a legacy issue solved.. I could link all the existing issues that revolve around this problem here but that would make this list too long for its own good)
- The ability to make wide area's of plains, mountain ranges, swamps, skylands or a combination of these whilst still being randomly selected. (imagine the large mountain strings in TerraForged)
- All of the suggested features here; https://github.com/PG85/OpenTerrainGenerator/issues/720 are possible with this one new feature added. (I am more then willing to make a preset for people to use that functions as base for the 4 possible modes )

Simply put Random mode gives the player a ever changing scenery to explore whereas from image gives the player a epic scenery of which the likes would not be possible in Random mode. Combine the two and a whole new world of possibilities open up.
